<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="50,50">
      <item>
       <widget class="QSplitter" name="splitter_13">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QSplitter" name="splitter_12">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QLineEdit" name="lineSetAddress">
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QPushButton" name="pushButtonCon">
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButtonDisc">
            <property name="text">
             <string>Disconnect</string>
            </property>
           </widget>
          </widget>
         </widget>
         <widget class="QSplitter" name="splitter_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QSplitter" name="splitter_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QLabel" name="labelMin">
            <property name="text">
             <string>Min</string>
            </property>
           </widget>
           <widget class="QSlider" name="horizontalSliderMin">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </widget>
          <widget class="QLCDNumber" name="lcdNumberMin"/>
         </widget>
         <widget class="QSplitter" name="splitter_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QSplitter" name="splitter_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QLabel" name="labelMax">
            <property name="text">
             <string>Max</string>
            </property>
           </widget>
           <widget class="QSlider" name="horizontalSliderMax">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </widget>
          <widget class="QLCDNumber" name="lcdNumberMax">
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </widget>
        </widget>
        <widget class="QSplitter" name="splitter_11">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QSplitter" name="splitter_10">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QSplitter" name="splitter_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QSplitter" name="splitter_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Timing(s)</string>
             </property>
            </widget>
            <widget class="QSlider" name="horizontalSliderTiming">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>3</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </widget>
          <widget class="QSplitter" name="splitter_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QPushButton" name="pushButtonStart">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButtonStop">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </widget>
         </widget>
         <widget class="QLabel" name="labelStatus">
          <property name="text">
           <string>Status</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditShowData"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>524</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberMin</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberMax</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderTiming</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
